psi4/src/psi4/libmints/mintshelper.cc

```diff
@@ -793,13 +797,13 @@ SharedMatrix MintsHelper::ao_helper(const std::string &label, std::shared_ptr<Tw
     return I;
 }
 
-SharedMatrix MintsHelper::ao_shell_getter(const std::string &label, std::shared_ptr<TwoBodyAOInt> ints, int M, int N,
-                                          int P, int Q) {
+SharedMatrix MintsHelper::ao_shell_getter(const std::string &label, std::shared_ptr <TwoBodyAOInt> ints, int M, int N, int P, int Q)
+{
     int mfxn = basisset_->shell(M).nfunction();
     int nfxn = basisset_->shell(N).nfunction();
     int pfxn = basisset_->shell(P).nfunction();
     int qfxn = basisset_->shell(Q).nfunction();
-    auto I = std::make_shared<Matrix>(label, mfxn * nfxn, pfxn * qfxn);
+    SharedMatrix I(new Matrix(label, mfxn * nfxn, pfxn * qfxn));
```

Comment:
It looks like you reversed a few of the `make_shared` changes. Can you look at patching that up?

Reply:
Indeed, the rebasing led to these. I will patch them up.